homeassistant/util/location.py

```diff
@@ -17,16 +17,35 @@
      'city', 'zip_code', 'time_zone', 'latitude', 'longitude',
      'use_fahrenheit'])
 
+DATA_SOURCE = ['https://freegeoip.io/json/', 'http://ip-api.com/json']
```

Comment:
https://freegeoip.io/ seems to be the successor of https://freegeoip.net/ and is pretty new (ref. https://github.com/fiorix/freegeoip/issues/165). Well, we don't know how stable they are or for how long they will be around either.